test/basic.js

```diff
@@ -81,6 +81,18 @@ describe('Basic Tests', () => {
       });
     });
 
+    it('Test getting training page as a normal user', () => {
+      const agent = request.agent(app);
+      return common.createNormalUser().then((member) => {
+        return common.createUserSession(member, agent).then(() => {
+          return agent
+            .get('/training')
+            .redirects(1)
```

Comment:
Should this redirect? Does it fail if you do not?

Most of the tests that are testing the correct level of access don't do a redirect unless they're a POST request

Reply:
No it is not necessary. Should I re commit or is that unnecessary?

Reply:
Yeah it should be removed if it isn't necessary